roles/docker/tasks/main.yml

```diff
@@ -102,7 +102,7 @@
     dest: /etc/sysconfig/docker
     regexp: '^OPTIONS=.*$'
     line: "OPTIONS='\
-      {% if ansible_selinux and ansible_selinux.status == '''enabled''' %} --selinux-enabled{% endif %}\
+      {% if docker_selinux_enabled is defined %} --selinux-enabled={{ docker_selinux_enabled }} {% elif ansible_selinux and ansible_selinux.status == '''enabled''' %} --selinux-enabled{% endif %}\
```

Comment:
@mtnbikenc hah, it looks like I previously started a review, but never submitted :)